src/libmusig2.h

```diff
@@ -27,12 +27,15 @@
  *              : state: The state of musig2.
  *              : nr_signers: The number of signers.
  * */
+ // todo: this should contain the 'signing' data, so maybe combined with the params.
 typedef struct{
     secp256k1_context* ctx;
     secp256k1_pubkey aggr_pk;
     secp256k1_pubkey aggr_R;
+    secp256k1_pubkey aggr_R_list[V];
+    int par_R;
+    int par_pk;
     unsigned char *L;
-    int state;
     int nr_signers;
 }musig2_context;
 
```

Comment:
We should consider rearranging the structs to avoid unnecessary parameter initialization for verification.